src/main/java/edu/harvard/hul/ois/fits/tools/mediainfo/MediaInfoUtil.java

```diff
@@ -272,11 +269,9 @@ protected Map<String, Map<String, String>> loadVideoDataMap(Map<String, String>
             String codecId = getMediaInfoString(ndx, "CodecID", MediaInfoNativeWrapper.StreamKind.Video);
             addDataToMap(videoTrackValuesMap, id, "codecId", codecId);
 
-            String codecCC = getMediaInfoString(ndx, "Codec/CC", MediaInfoNativeWrapper.StreamKind.Video);
-            if (codecCC != null) {
-                codecCC = codecCC.trim();
+            if (codecId.length() == 4) {
```

Comment:
An inline comment would be helpful here to:
- describe the meaning of the magic number `4`, and
- describe why we are setting the "codecCC" value to 'codecId'

Reply:
done